docs/docs/sourcing-from-agilitycms.md

```diff
@@ -0,0 +1,89 @@
+## Getting Started
+
+### Create A Free Agility Account
+Create an Agility CMS account with the Free Plan (this plan is free forever). [Sign up to Agility CMS](https://account.agilitycms.com/sign-up?product=agility-free).  
+
+Once your account is created, we'll need to grab your GUID and API Keys.
+
+### Get the Code
+
+Make sure you have Gatsby CLI tools installed (we are using npm here...)
+```shell
+npm install -g gatsby-cli
+```
+
+Go ahead and clone the [Agility CMS Gatsby Starter](https://github.com/agility/agility-gatsby-starter) repo on from GitHub that has all the code you need to get started.
+```shell
+git clone https://github.com/agility/agility-gatsby-starter.git
+```
+
+Resolve any dependencies
+
+```shell
+npm install
+```
+
+Run it in development mode!
+```shell
+gatsby develop
+```
+
+The site is just a starter, but it has a bunch of interesting features that you can use to build from.  Let's hook this code up to your new Agility CMS instance that you just created.
+
+## Hook it up to your Agility CMS instance
+Edit the `gatsby-config.js` file and repace the `guid` and `apiKey` with yours.  
+
+You can find your API keys on the Getting Started page in the Agility CMS Content Manager.
+
+
+![Agility CMS - Dashboard - API Keys](./images/agilitycms-api-keys.png)
+
+If you use the `preview` key, you won't have to publish to see the changes you've made show up. If you use the `fetch` key, make sure you've published any content you wish to see changed.
+
+## How Does It Work
+
+The Gatsby Source Plugin downloads all the Pages on the Agility CMS Sitemap, as well as any Shared Content that's referenced on the `sharedContent` property on of the `gatsby-config.js` file.
+
+All of those pages and content are then made available in GraphQL to the React Components you will write to render those pages.
+
+Check out the component called "Jumbotron".  This is a simple example of how to display a styled heading and sub-heading with content that comes from Agility CMS.  Here is the Module that provides this content being edited in the Agility CMS Content Manager:
+
+![Agility CMS - Example Module - Jumbotron](./images/agilitycms-jumbotron.png)
+
+
+And here is the code used to render it.  Notice that the `title` and `subTitle` fields are available as properties the `item.fields` object.
+
+```javascript:title=/src/modules/Jumbotron.js
+import React, { Component } from 'react';
+import { graphql, StaticQuery } from "gatsby"
+
+import './Jumbotron.css'
+
+export default class Jumbotron extends Component {
+    render() {    
+        return (
+            <section className="jumbotron">
+                <h1>{this.props.item.fields.title}</h1>
+                <h2>{this.props.item.fields.subTitle}</h2>
+            </section>
+        );
+    }
+}
+```
+
+When we add new modules and content definitions to Agility CMS, the components we use to render those modules will automatically get the strongly typed data delivered to those modules as props.  
```

Comment:
```suggestion
When you add new modules and content definitions to Agility CMS, the components used to render those modules will automatically receive the strongly typed data delivered to those modules as props.  
```

A conclusion with where to go for further information would be good to wrap up this doc!